specification/common-types/resource-management/v5/networksecurityperimeter.json

```diff
@@ -0,0 +1,410 @@
+          "type": "array",
+          "readOnly": true,
+          "items": {
+            "description": "Fully qualified resource ID of the suggested resource",
+            "type": "string",
+            "format": "arm-id",
+            "readOnly": true
+          }
+        },
+        "suggestedAccessRules": {
+          "description": "Access rules that can be added to the same profile to remediate the issue.",
+          "type": "array",
+          "readOnly": true,
+          "items": {
+            "$ref": "#/definitions/AccessRule"
+          },
+          "x-ms-identifiers": []
+        }
+      }
+    },
+    "NetworkSecurityPerimeter": {
+      "description": "Information about a network security perimeter (NSP)",
+      "type": "object",
+      "properties": {
+        "id": {
+          "description": "Fully qualified Azure resource ID of the NSP resource",
+          "type": "string",
+          "format": "arm-id",
+          "x-ms-arm-id-details": {
+            "allowedResources": [
+              {
+                "resourceType": "Microsoft.Network/networkSecurityPerimeters"
+              }
+            ]
+          }
+        },
+        "perimeterGuid": {
+          "description": "Universal unique ID (UUID) of the network security perimeter",
+          "type": "string",
+          "format": "uuid"
+        },
+        "location": {
+          "description": "Location of the network security perimeter",
+          "type": "string",
+          "x-ms-mutability": [
+            "create",
+            "read"
+          ]
+        }
+      }
+    },
+    "ResourceAssociationAccessMode": {
+      "description": "Access mode of the resource association",
+      "enum": [
+        "Enforced",
+        "Learning",
+        "Audit"
+      ],
+      "type": "string",
+      "x-ms-enum": {
+        "name": "ResourceAssociationAccessMode",
+        "modelAsString": true,
+        "values": [
+          {
+            "value": "Enforced",
+            "description": "Enforced access mode - traffic to the resource that failed access checks is blocked"
+          },
+          {
+            "value": "Learning",
+            "description": "Learning access mode - traffic to the resource is enabled for analysis but not blocked"
+          },
+          {
+            "value": "Audit",
+            "description": "Audit access mode - traffic to the resource that fails access checks is logged but not blocked"
+          }
+        ]
+      }
+    },
+    "ResourceAssociation": {
+      "description": "Information about resource association",
+      "type": "object",
+      "properties": {
+        "name": {
+          "description": "Name of the resource association",
+          "type": "string"
+        },
+        "accessMode": {
+          "$ref": "#/definitions/ResourceAssociationAccessMode"
+        }
+      }
+    },
+    "NetworkSecurityProfile": {
```

Comment:
"diagnosticSettingsVersion" and "enabledLogCategories" are missing from this object.

Reply:
Fixed in https://github.com/Azure/azure-rest-api-specs/pull/29117